### DISCUSSION TITLE
考虑做全平台吗，移动端的ios跟android

### DISCUSSION THREAD

**Discussion by Sheep2706** [clash-verge-rev/clash-verge-rev#1857] [Ideas]
希望前辈考虑移动端

**Comment by wonfen** [comment]
未来有计划。

**Comment by MITchell-eng** [comment]
安卓的好像有吧

**Reply by Sheep2706** [reply]
安卓没有啊 release就没有

**Comment by TravelTibet** [comment]
安卓有`Clash Meta for Android`😥

**Reply by Sheep2706** [reply]
瞎说

**Reply by TravelTibet** [reply]
> 瞎说

![Screenshot_20250125_210021_Clash Meta for Android.jpg](https://github.com/user-attachments/assets/5b4a1d6a-2997-497b-82e0-10417cfe5577)

**Reply by Sheep2706** [reply]
这只是原clash fork出来的

**Reply by Love-in-idleness** [reply]
> 这只是原clash fork出来的

但这个是mihomo内核开发者维护的，已经换了新内核了，完全能用啊。再者还有全平台的Hiddify。做移动端不如选择把桌面端做得更好。

**Comment by A81N9** [comment]
期待中...
目前 Android 端还没找到任何一个支持扩展脚本功能
